addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -1836,3 +1836,68 @@ QUnit.test("Escape key should focus the composer if it's not focused", async ()
     triggerHotkey("escape");
     await contains(".o-mail-Composer-input:focus");
 });
+
+QUnit.test("Notification settings: basic rendering", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "Mario Party",
+        channel_type: "channel",
+    });
+    const { openDiscuss } = await start();
+    openDiscuss(channelId);
+    await click(".o-mail-Discuss-header button[title='Notification Settings']");
+    await contains(".o-discuss-NotificationSettings button", { text: "All Messages" });
+    await contains(".o-discuss-NotificationSettings button", { text: "Mentions Only" });
+    await contains(".o-discuss-NotificationSettings button", { text: "Nothing" });
+
+    await click(".o-discuss-NotificationSettings button[title='Mute Channel']");
+    await contains(".dropdown-item[title='For 15 minutes']");
+    await contains(".dropdown-item[title='For 1 hour']");
+    await contains(".dropdown-item[title='For 3 hours']");
+    await contains(".dropdown-item[title='For 8 hours']");
+    await contains(".dropdown-item[title='For 24 hours']");
+    await contains(".dropdown-item[title='Until I turn it back on']");
```

Comment:
- Useless empty line => remove it (keep code of a test cohesive, empty lines gives impression these are 2 tests rather than one)
- `o-mail-Discuss-header button` is useless noise, I would just grep from title attribute (functionally relevant enough). Same for `dropdown-item` which is not that relevant and just adds noise to the readability of test.